sys/include/net/gcoap/forward_proxy.h

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (C) 2020 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for
+ * more details.
+ */
+
+/**
+ * @defgroup    net_gcoap_forward_proxy    Gcoap Forward Proxy
+ * @ingroup     net_gcoap
+ * @brief       Forward proxy implementation for Gcoap
+ *
```

Comment:
Maybe add a note here
```suggestion
 * @note Does not support CoAPS yet.
```